website/pages/en/deploying/deploying-a-subgraph-to-studio.mdx

```diff
@@ -13,7 +13,7 @@ These are the steps to deploy your subgraph to the Subgraph Studio:
 
 ## Installing Graph CLI
 
-We are using the same CLI to deploy subgraphs to our [hosted service](https://thegraph.com/hosted-service/) and to the [Subgraph Studio](https://thegraph.com/studio/). Here are the commands to install graph-cli. This can be done using npm or yarn.
+We are using the same CLI to deploy subgraphs to the [Subgraph Studio](https://thegraph.com/studio/). Here are the commands to install graph-cli. This can be done using npm or yarn.
```

Comment:
```suggestion
We are using the same CLI to deploy subgraphs to the [Subgraph Studio](https://thegraph.com/studio/). Here are the commands to install `graph-cli`. This can be done using `npm` or `yarn`.
```